lib/_internal/file_picker_web.dart

```diff
@@ -185,6 +186,47 @@ class FilePickerWeb extends FilePicker {
     return files == null ? null : FilePickerResult(files);
   }
 
+  @override
+  Future<String?> saveFile({
+    String? dialogTitle,
+    String? fileName,
+    String? initialDirectory,
+    FileType type = FileType.any,
+    List<String>? allowedExtensions,
+    Uint8List? bytes,
+    bool lockParentWindow = false,
+  }) async {
+    if (bytes == null) {
+      throw ArgumentError(
+        'The bytes are required when saving a file on the web.',
+      );
+    }
+
+    if (fileName == null || fileName.isEmpty) {
+      throw ArgumentError(
+          'A file name is required when saving a file on the web.');
```

Comment:
There is a missing comma, which will cause a formatting diff.
Once we switch to Dart 3.7, any missing comma's will be inserted automatically, but for now we need to do that manually.

```suggestion
          'A file name is required when saving a file on the web.',
       );
```